### PR TITLE
plugin.audio.pureradio] 1.0.0

### PR DESCRIPTION
**Pull request by bdommel**
### Description
This plugin enables the playback of the PureRadio audio stream, which can be set in different qualities using different bitrates. Also a selection of podcast, hosted on Spreaker, are available for instant playback.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.audio.pureradio/addon.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		<summary lang="en">PureRadio.One.</summary>
+		<description lang="en">PureRadio is an internet station with just good music. We love music and like to share it with our listeners.<br /><br />Tune in, join us on Facebook and get involved.</description>
+		<disclaimer lang="en"></disclaimer>
+		<language>en</language>
+		<platform>all</platform>
+		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+		<forum></forum>
+		<website>http://pureradio.one</website>
+		<email>pureradio.one@gmail.com</email>
+		<source></source>
```
Comment (phil65) [inline plugin.audio.pureradio/addon.xml]: plase add source link.

File: plugin.audio.pureradio/addon.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.pureradio" name="PureRadio.One" version="1.0.0" provider-name="cubitus">
+	<requires>
+		<import addon="xbmc.python" version="2.1.0"/>
```
Comment (phil65) [inline plugin.audio.pureradio/addon.xml]: 2.24.0

File: plugin.audio.pureradio/default.py [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+URL128   = 'http://listento.pureradio.one:8000/pure_128'
+URL192   = 'http://listento.pureradio.one:8000/pure_192'
+PODCASTS = 'http://www.spreaker.com/show/{0}/episodes/feed'
+
+ADDON    = xbmcaddon.Addon(ADDONID)
+HOME     = ADDON.getAddonInfo('path')
+VERSION  = ADDON.getAddonInfo('version')
+GETTEXT  = ADDON.getLocalizedString
+
+ICON     =  os.path.join(HOME, 'icon.png')
```
Comment (phil65) [inline plugin.audio.pureradio/default.py]: getAddonInfo("icon")

File: plugin.audio.pureradio/default.py [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+FANART   =  os.path.join(HOME, 'fanart.jpg')
```
Comment (phil65) [inline plugin.audio.pureradio/default.py]: getAddonInfo("fanart")

**Comment by bdommel** [comment]
I included the review comments and commited a new version "Correcting for PR comments"

**Comment by phil65** [comment]
looks good to me now. Would be cool if you could convert the language xmls to po files somewhen during your next PRs.
Thanks!

**Comment by bdommel** [comment]
Hi Phil, the next PR will have PO files. Many thanks for your help!

**Comment by bdommel** [comment]
Used strings.po instead of strings.xml.

**Comment by bdommel** [comment]
squashed commits

**Review by razzeee — APPROVED** [review]

**Comment by Razzeee** [comment]
LGTM

File: plugin.audio.pureradio/addon.xml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+<addon id="plugin.audio.pureradio" name="PureRadio.One" version="1.0.0" provider-name="cubitus">
+	<requires>
+		<import addon="xbmc.python" version="2.24.0"/>
+	</requires>
+	<extension point="xbmc.python.pluginsource" library="default.py">
+		<provides>audio</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en">PureRadio.One.</summary>
+		<description lang="en">PureRadio is an internet station with just good music. We love music and like to share it with our listeners.<br /><br />Tune in, join us on Facebook and get involved.</description>
```
Comment (phil65) [inline plugin.audio.pureradio/addon.xml]: oh, overlooked this. Please use `[CR]` for line breaks, not `<br />`

**Pull request merged**